help_to_heat/frontdoor/views.py

```diff
@@ -371,6 +375,13 @@ def get(self, request, session_id, page_name, unsaved_data=None, errors=None, is
         if data_with_get_answers != data:
             save_answer(session_id, page_name, data_with_get_answers)
 
+        if page_name not in schemas.routing_overrides:
+            try:
+                # ensure the user has answers to complete the journey to this page
+                calculate_journey(answers, to_page=page_name)
+            except CouldNotCalculateJourneyException:
+                return redirect("/sorry-journey")
```

Comment:
Do we want to log a warning or something else below an error here now? I forget whether that's covered by another ticket.

Reply:
good point; we do get the choice here for whether we want to or not

I'd err on doing so for now (I'll update the code), then we'll monitor going forwards whether we still want to or not (covered by that other ticket)